packages/lodestar-cli/src/commands/beacon.ts

```diff
@@ -33,6 +43,11 @@ export class BeaconNodeCommand implements ICliCommand {
       .command("beacon")
       .description("Start lodestar node")
       .option("-c, --configFile [config_file]", "Config file path")
+      .option("-p, --preset [preset]", "Minimal/mainnet", "minimal")
+      .option("-s, --genesisState [params]", "Start chain from known state")
+      .option("--peer-id [peerId]", "peer id hex string or json file path")
```

Comment:
we will likely only support the json file path option here (we need the private key as well as the public key)

Reply:
so I just need to modify the option description or what else I have to do?

Reply:
instead of using `getPeerId`, L75, use `loadPeerIdFromJsonFile`